plugins/yaml-schema/tests/test-server/server.ts

```diff
@@ -0,0 +1,13 @@
+export const startServer = (app, port, data) => {
```

Comment:
We're using this in a couple packages now. I think it should be moved to the `utils/testing` package